### PR TITLE
torch_histc

### PR DESCRIPTION
**Pull request by teckno**

### CODE REVIEW

File: ivy/functional/frontends/torch/miscellaneous_ops.py [hunk 1]
```diff
@@ -381,3 +381,8 @@ def diag(input, diagonal=0, *, out=None):
 @to_ivy_arrays_and_back
 def clone(input):
     return ivy.copy_array(input)
+
+
+@to_ivy_arrays_and_back
+def histc(input, bins=10, min=None, max=None, *, out=None):
+    return ivy.histc(input, bins=bins, min=min, max=max, out=out)
```
[the remark this answers is not in the document]
Comment (teckno) [reply ivy/functional/frontends/torch/miscellaneous_ops.py:388]: I guess it's a compositional function again ?

File: ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py [hunk 2]
```diff
@@ -1318,3 +1318,50 @@ def test_torch_clone(
         on_device=on_device,
         input=x[0],
     )
+
+# histc
+
+
+@handle_frontend_test(
+    fn_tree="ivy.histc",
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py:1326]: The function tree should be for torch

File: ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py [hunk 3]
```diff
@@ -1318,3 +1318,50 @@ def test_torch_clone(
...
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        shape=st.shared(helpers.get_shape(min_num_dims=1, max_num_dims=2), key="shape"),
+        min_num_dims=1,
+    ),
+    bins=st.integers(min_value=1, max_value=100),
+    min=st.integers(min_value=-100, max_value=100),
+    max=st.integers(min_value=-100, max_value=100),
+    out=st.just(None),
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py:1335]: No need to specify out if we have just generated `test_without`

File: ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py [hunk 4]
```diff
@@ -1318,3 +1318,50 @@ def test_torch_clone(
...
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input=x[0],
+        bins=bins,
+        min=min,
+        max=max,
+        out=out,
+        test_with_out=test_with_out,
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py:1365]: These should not be added here as they're included in test_flags

Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_miscellaneous_ops.py:1365]: These should not be added here as they're included in test_flags

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @teckno 

Below are a few comments for changes needed😊

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**